### DISCUSSION TITLE
DNS check error when hypen exist in domain name.

### DISCUSSION THREAD

**Discussion by onurbektas** [postalserver/postal#2681] [Help with using Postal] [answered]
Hi, 

We are getting DNS check  error if hypen (i.e acme-sample.com) exists in domain name.  Anybody experienced this before ? I tried the dig the code and found  https://github.com/postalserver/postal/blob/main/app/models/track_domain.rb#L30 but I am not a ruby guy. As far as I can understand, it  checks domain name against regex format: {/\A[a-z0-9-]+\z/ }

**Comment by airshell** [comment]
Hi @onurbektas  It should not be an issue if your domain name contain hypen "-" like **goo-gle.com**.

My previous domain was also having hypen in the domain name and all the DNS records were validated properly by Postal.

**Comment by willpower232** [comment, marked as answer]
Are you able to share a screenshot of the error you're seeing?

It may just be that you need to change this setting:

https://github.com/postalserver/postal/blob/2f62baa238fc1102706ee4acf079b7a876b05283/config/postal.defaults.yml#L17

**Reply by onurbektas** [reply]
Setting 'use_local_ns_for_domains: true' solved the problem, thank you.
